CHANGELOG.md

```diff
@@ -97,10 +98,10 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#4123](https://github.com/thanos-io/thanos/pull/4123) Query: match external labels for exemplars API.
 
 ### Changed
-- 
+-
```

Comment:
Do we need this?

Reply:
@bwplotka Can we edit the CHANGELOG.md file using Prettier? This will help to remove indentations.

Reply:
@bwplotka Can we edit the `CHANGELOG.md` file using Prettier? This will help to remove indentations.

Reply:
@iamrajiv simply running `make docs` from the root of the repo, would work.

Reply:
Ah no, if it was done my out tool `make docs` then fine. Thanks! See that build is failing now because it's lacking this =D

Reply:
I think it would be easier to just remove the last two commits! [The revert commit](https://github.com/thanos-io/thanos/pull/4572/commits/4b62d7a7299c1beb83606be12ec9d3b1c8c07a3f) removes the changelog entry but does not add back the spaces removed [here](https://github.com/thanos-io/thanos/pull/4572/commits/42becc74a620f970b9755d98ef81a493ae4842bc). Or just running `make docs` once would be sufficient. 🙂